packages/picker/test/index.ts

```diff
@@ -1123,14 +1134,20 @@ export function runPickerTests(): void {
                 input2.remove();
             });
             it('tabs forward through the element', async () => {
-                // start at input1
-                input1.focus();
-                await nextFrame();
-                expect(document.activeElement === input1, 'focuses input 1').to
-                    .true;
-                // tab to the picker
-                let focused = oneEvent(el, 'focus');
-                await sendKeys({ press: 'Tab' });
+                let focused: Promise<CustomEvent<FocusEvent>>;
+                if (!isSafari) {
```

Comment:
Safari should only have Tab focusable elements if the user preferences are configured to do so. We should not force something contrary to suer preferences.